overview/glossary.md

```diff
@@ -0,0 +1,42 @@
+| Identity                 | An identity in uPort is really just someone or something that can sign data or transactions and also receive signed data about itself.  Essentially it is an address on a decentralized network, controlled by a private signing key |
+| Issuer                   | A reference to the identity of an application or user that issued a signed JWT                                                                                                                                                       |
+| muPort(Simple ID)        | A way without a Proxy contract to create, update, and recover uPort identities in addition to encryption and decryption of data related to the identity                                                                                                                                                              |
+| [muPort-DID-Resolver](https://github.com/uport-project/muport-did-resolver)    | A DID resolver for muPort/SimpleID resolution with support for creating identities.                                                                                              |
+| PKI                      | Public Key Infrastructure.  uPort implements a simple PKI to verify off-chain messages                                                                                                                                               |
+| Public Key               |  Used to verify privately signed data                                                                                                                                                                                                |
+| Private Key              |  Used to sign data and transactions                                                                                                                                                                                                  |
+| Recovery Seed            |  A 12-word pass phrase used to retrieve the private keys of an Identity                                                                                                                                                              |
+| [Secp256k1-DID-Resolver](https://github.com/uport-project/secp256k1-did-resolver) | A DID resolver that uses secp256k1 public keys as decentralized identifiers, as oppose to an ethereum address.                                                                                     |
+| Selective Disclosure     | A request and response for specific data about claims (attestations) related to an decentralized identity.                                                                                                                            |
+| Segregated Account       | A key derived from the seed of a root identity to be used instead of the root key to avoid correlating blockchain activity with the root key.                                                                                        |
```

Comment:
An ethereum account specifically used for interactions between your app and your user. It is called segregated as it for privacy reasons segregates the transactions to a single app, making it harder to correlate who the user is for external parties